apps/web/pages/getting-started.tsx

```diff
@@ -371,8 +383,8 @@ export default function Onboarding(props: inferSSRProps<typeof getServerSideProp
                 <TimezoneSelect
                   id="timeZone"
                   value={selectedTimeZone}
-                  onChange={({ value }) => setSelectedTimeZone(value)}
-                  className="mt-1 block w-full rounded-md border-gray-300 text-sm focus:border-blue-500 focus:ring-blue-500"
+                  onChange={({ value }: { value: string }) => setSelectedTimeZone(value)}
```

Comment:
```suggestion
                  onChange={({ value }) => setSelectedTimeZone(value)}
```

This should be inferred AFAIK